Mathlib/Algebra/Order/Field/Power.lean

```diff
@@ -119,6 +119,11 @@ section LinearOrderedField
 
 variable [LinearOrderedField α] {a b c d : α} {n : ℤ}
 
+lemma one_le_sq_div_abs_sq (ha : a ≠ 0) : 1 ≤ a^2 / |a|^2 := by
+  rw [_root_.sq_abs, le_div_iff']
+  simp only [mul_one, le_refl]
+  exact (sq_pos_iff a).mpr ha
```

Comment:
Isn't this true simply because `|a| ^ 2 = a ^ 2`?

Reply:
oops yes, it should be `=1` as well. Let me fix that. Do you think its worth having as a lemma? `apply?` couldn't find it, but maybe its already somewhere

Reply:
No, it's provable by simp (hopefully) and is a straightforward combination of easier lemmas. So no need.

Reply:
simp doesn't prove it for me, which is why I had added it.

Reply:
`simp [ha]` really should prove it

Reply:
Ok I figured out why it wouldn't simp for me. It seems

```
import Mathlib

open Complex

lemma test (a : ℤ) (ha : a ≠ 0) : (a : ℝ)^2/|(a : ℝ)|^2=1 := by simp [ha]
```
doesn't work but

```
import Mathlib


lemma test (a : ℤ) (ha : a ≠ 0) : (a : ℝ)^2/|(a : ℝ)|^2=1 := by simp [ha]
```

does work, since it doesn't get confused about which absolute value to use.

Reply:
Ah yeah, that's very unfortunate. We should fix this.

Reply:
`Complex.int_cast_abs` seems weird to me, I would've thought it would be the other way for a simp. This seems to be what causes the confusion. Or one of the things.

Reply:
Agreed. This one is really strange. You shouldn't have to involve complex numbers from a goal solely about real numbers!

Fixed in #10543

Reply:
You're cleared! It was @urkud in https://github.com/leanprover-community/mathlib/pull/5423.